py2app/recipes/black.py

```diff
@@ -0,0 +1,25 @@
+from pkg_resources import get_distribution
+from pathlib import Path
+
+
+def check(cmd, mf):
+    m = mf.findNode("black")
+    if m is None or m.filename is None:
+        return None
+
+    egg = Path(get_distribution('black').egg_info)
+    top = egg / 'top_level.txt'
+
+    # These cannot be in zip
+    packages = ["black", "blib2to3"]
```

Comment:
Why can't these be in a zipfile? The code doesn't seem to use ``__file__`` (which is a telltale sign for code that won't work in zipfiles), other than to determine if it is running as a compiled C extension.

Reply:
Let me verify and report back the issues.